_includes/sidenav.html

```diff
@@ -24,16 +23,17 @@
 
 <nav class="nav-secondary expandable {{ additional_classes }}">
     <h1 class="u-visually-hidden">
-    {%- for href, id, caption in items %}
+    {%- for item in items %}
```

Comment:
Hmm, yeah, that's probably going to be harder, since it could be any of those top-level pages. Could we check to see if it's a child/sub-page first, then if it's not, grab the page title? If it is, grab the parent page title?